Community/Tdarr_Plugin_x7ac_Remove_Closed_Captions.js

```diff
@@ -1,62 +1,54 @@
-/* eslint-disable */
 function details() {
   return {
-    id: "Tdarr_Plugin_x7ac_Remove_Closed_Captions",
-    Stage: "Pre-processing",
-    Name: "Remove closed captions",
-    Type: "Video",
-    Operation: "Remux",
+    id: 'Tdarr_Plugin_WS_Remove_Closed_Captions',
```

Comment:
Plus, the ID of the plugin doesn't change.

Reply:
Unintended change. Thanks!